apps/src/codebridge/FileBrowser/FileBrowserRow/hooks/useFileRowOptions.ts

```diff
@@ -90,6 +94,25 @@ export const useFileRowOptions = (
         labelText: codebridgeI18n.renameFile(),
         clickHandler: () => openRenameFilePrompt({fileId: file.id}),
       },
+      {
+        condition: enableUserAddedContext(appName, file),
+        iconName: 'message-code',
+        labelText: codebridgeI18n.addToAiTutorContext(),
+        clickHandler: () => {
+          const folderPath = getFolderPath(file.folderId, projectFolders);
+          let fullFilename = file.name;
+          if (folderPath !== '/') {
+            fullFilename = (folderPath + '/' + file.name).substring(1); // remove leading slash
+          }
+          dispatch(
+            addItemToUserAddedContext({
+              displayName: fullFilename,
```

Comment:
Curious why you have both `displayName` and `filename` - they seem like duplicates.